src/plat/pc99/machine/hardware.c

```diff
@@ -76,6 +76,10 @@ BOOT_CODE static inline uint32_t measure_tsc_khz(void)
 BOOT_CODE uint32_t tsc_init(void)
 {
 
+#if CONFIG_PC99_TSC_FREQUENCY > 0
+    return CONFIG_PC99_TSC_FREQUENCY / 1000000;
```

Comment:
worth saying that the CONFIG_PC99_TSC_FREQUENCY is in the unit of HZ, and the return value is MHz.

Reply:
I updated the config help string to say that the unit is Hz